extensions/amp-dynamic-css-classes/0.1/amp-dynamic-css-classes.js

```diff
@@ -48,6 +66,30 @@ export function referrers_(referrer) {
 }
 
 /**
+ * Normalizes certain referrers across devices.
+ * @param {!Window} win
+ * @returns {!Array<string>}
+ */
+function normalizedReferrers(win) {
+  const referrer = referrerDomain(win);
+  let referrers = referrers_(referrer);
+
+  // Normalize t.co names to twitter.com
+  if (referrer === 't.co') {
+    referrers = referrers.concat(referrers_('twitter.com'));
```

Comment:
Don't you think that'll be confusing for the people who are expecting `t.co` links?